app/model/orm/measurement_context.py

```diff
@@ -87,21 +87,23 @@ def get_df(self, db_session):
         return execute_into_df(db_session, query)
 
     def get_chart_label(self, db_session):
+        from markupsafe import Markup, escape
+
         subject      = self.get_subject(db_session)
         technique    = self.technique
         bioreplicate = self.bioreplicate
         compartment  = self.compartment
         experiment   = bioreplicate.experiment
 
         if technique.subjectType == 'metabolite':
-            label_parts = [f"<b>{subject.name}</b>"]
+            label_parts = [f"<b>{escape(subject.name)}</b>"]
         else:
-            label_parts = [technique.short_name]
+            label_parts = [escape(technique.short_name)]
```

Comment:
It's important to html-escape values that may come from the user. The short name now includes a label that is written down by the user so we need to be careful.